command/v7/shared/app_summary_displayer_test.go

```diff
@@ -696,126 +750,245 @@ var _ = Describe("app summary displayer", func() {
+									StatusValue:      constant.DeploymentStatusValueActive,
+									StatusReason:     constant.DeploymentStatusReasonDeploying,
+									LastStatusChange: "",
+									Options: resources.DeploymentOpts{
+										MaxInFlight: maxInFlightDefaultValue,
+									},
+								},
+							}
+						})
+
+						It("displays the message", func() {
+							Expect(testUI.Out).To(Say(`Rolling deployment currently DEPLOYING`))
+							Expect(testUI.Out).NotTo(Say(`\(since`))
+						})
+						It("does not display max-in-flight", func() {
+							Expect(testUI.Out).NotTo(Say(`max-in-flight`))
+						})
 					})
 				})
 
 				When("the deployment is cancelled", func() {
-					BeforeEach(func() {
-						summary = v7action.DetailedApplicationSummary{
-							Deployment: resources.Deployment{
-								Strategy:         constant.DeploymentStrategyRolling,
-								StatusValue:      constant.DeploymentStatusValueActive,
-								StatusReason:     constant.DeploymentStatusReasonCanceling,
-								LastStatusChange: LastStatusChangeTimeString,
-							},
-						}
+					When("max-in-flight value is non-default", func() {
+						BeforeEach(func() {
+							summary = v7action.DetailedApplicationSummary{
+								Deployment: resources.Deployment{
+									Strategy:         constant.DeploymentStrategyRolling,
+									StatusValue:      constant.DeploymentStatusValueActive,
+									StatusReason:     constant.DeploymentStatusReasonCanceling,
+									LastStatusChange: LastStatusChangeTimeString,
+									Options: resources.DeploymentOpts{
+										MaxInFlight: 2,
+									},
+								},
+							}
+						})
+
+						It("displays the message", func() {
+							var actualOut = fmt.Sprintf("%s", testUI.Out)
+							Expect(actualOut).To(MatchRegexp(`Rolling deployment currently CANCELING \(since %s\)`, dateTimeRegexPattern))
+						})
+						It("displays max-in-flight value", func() {
+							Expect(testUI.Out).To(Say(`max-in-flight: 2`))
+						})
 					})
+					When("max-in-flight value is default", func() {
+						BeforeEach(func() {
+							summary = v7action.DetailedApplicationSummary{
+								Deployment: resources.Deployment{
+									Strategy:         constant.DeploymentStrategyRolling,
+									StatusValue:      constant.DeploymentStatusValueActive,
+									StatusReason:     constant.DeploymentStatusReasonCanceling,
+									LastStatusChange: LastStatusChangeTimeString,
+									Options: resources.DeploymentOpts{
+										MaxInFlight: maxInFlightDefaultValue,
+									},
+								},
+							}
+						})
 
-					It("displays the message", func() {
-						var actualOut = fmt.Sprintf("%s", testUI.Out)
-						Expect(actualOut).To(MatchRegexp(`Rolling deployment currently CANCELING \(since %s\)`, dateTimeRegexPattern))
+						It("displays the message", func() {
+							var actualOut = fmt.Sprintf("%s", testUI.Out)
+							Expect(actualOut).To(MatchRegexp(`Rolling deployment currently CANCELING \(since %s\)`, dateTimeRegexPattern))
```

Comment:
*question:* I thought that Say supported Regexes. Any particular reason for not using it here?

Reply:
Addressed in latest commit